source/detect-version.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+const inquirer = require('inquirer');
+const version = require('./version');
+const git = require('./git-util');
+const getCommits = require('./get-commits');
+const printCommitLog = require('./print-commit-log');
+
+const listCommits = commits => {
+	return commits.map(commit => `- ${commit.message}  ${commit.id}`);
+};
+
+const generateReleaseNotes = (commits, repoUrl, latest, nextTag) => {
+	const output = [];
+
+	const majorChanges = commits.filter(commit => commit.type === 'major');
+	const minorChanges = commits.filter(commit => commit.type === 'minor');
+	const patchChanges = commits.filter(commit => commit.type === 'patch');
+
+	if (majorChanges.length > 0) {
+		output.push('## Breaking changes', '', ...listCommits(majorChanges), '');
```

Comment:
```suggestion
		output.push('### Breaking', '', ...listCommits(majorChanges), '');
```

i think `###` looks nicer. That's what I've been using.